products/arangodb

```diff
@@ -0,0 +1,33 @@
+---
+title: ArangoDB
+category: db
+permalink: /arangodb
+releasePolicyLink: https://arangodb.com/master-services-agreement-inc-may-2023-cs/#eolpolicy
+changelogTemplate: "https://docs.arangodb.com/stable/release-notes/version-__RELEASE_CYCLE__/"
+releaseLabel: "ArangoDB __RELEASE_CYCLE__"
+eolWarnThreshold: 121
+releases:
+-   releaseCycle: "3.10"
+    latest: "3.10.11"
+    releaseDate: 2022-10-04
+    eol: 2024-04-15
+-   releaseCycle: "3.11"
+    latest: "3.11.6"
+    releaseDate: 2023-05-30
+    eol: 2024-11-30
+-   releaseCycle: "3.12"
+    latest: "unreleased"
+    releaseDate: false
+    eol: false
+
+---
+
+> [ArangoDB](https://arangodb.com/) is a multi-model graph and document database.
+
+Major releases are supported for 18 months and minor releases for 12 months following release, per the [EOL policy](https://arangodb.com/master-services-agreement-inc-may-2023-cs/#eolpolicy).
```

Comment:
```suggestion
Major releases are supported for at least 18 months. Minor releases for 12 months from their general availability.
```

This looks a bit clearer to me, WDYT ?

It could be great to also document what does support means, if it's documented somewhere on https://arangodb.com.

And no need to repeat the link, a sentence will be automatically generated from the `releasePolicyLink`.